### PR TITLE
python3Packages.twinkly-client: init at 0.0.2

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Python module to communicate with Twinkly LED strings

https://github.com/dr1rrb/py-twinkly-client

This is a Home Assistant dependency.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 113785` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>python37Packages.twinkly-client</li>
    <li>python38Packages.twinkly-client</li>
    <li>python39Packages.twinkly-client</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 113785` at 2d7784c9 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/829508c816cebb5992896e9090cf21d5">python37Packages.twinkly-client</a></li></li>
    <li><a href="https://gist.github.com/e0bd529dce49c76cbf75b8fabc37648a">python38Packages.twinkly-client</a></li></li>
    <li><a href="https://gist.github.com/0f8d716343f077d6abb02c87b4ceb08e">python39Packages.twinkly-client</a></li></li>
  </ul>
</details>

File: pkgs/development/python-modules/twinkly-client/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "16jbm4ya4yk2nfswza1kpgks70rmy5lpsv9dv3hdjdnr1j44hr3i";
+  };
+
+  propagatedBuildInputs = [ aiohttp ];
+
+  # Projec has no tests
```
Comment (SCOTT-HAMILTON) [inline pkgs/development/python-modules/twinkly-client/default.nix:20]: Typo

Comment (fabaff) [reply pkgs/development/python-modules/twinkly-client/default.nix]: Thanks, fixed.

**Comment by SuperSandro2000** [comment]
@fabaff please resolve the merge conflict.

**Comment by fabaff** [comment]
> @fabaff please resolve the merge conflict.

@SuperSandro2000, done.

**Review by mweinelt — APPROVED** [review]

**Pull request merged**